.github/workflows/handle_stalled.yml

```
@@ -15,20 +15,22 @@ jobs:
           days-before-stale: 20
           days-before-close: 10
 
-          exempt-issue-labels: "bot/ignore"
-          stale-issue-label: "meta/stale"
+          exempt-issue-labels: stale-bot/ignore
+          stale-issue-label: meta/stale
           stale-issue-message: >
             This issue has become stale because it has been open for 20 days without
-            activity. Remove the label and comment or this issue will be closed in 10 days.
+            activity. Remove the label (maintainer) or comment on this issue (assignee).
+            Otherwise, this issue will be closed in 10 days automatically.
           close-issue-label: "meta/closed due to age or inactivity"
           close-issue-message: >
             This issue was closed due to inactivity.
 
-          exempt-pr-labels: "bot/ignore"
-          stale-pr-label: "meta/stale"
+          exempt-pr-labels: stale-bot/ignore
+          stale-pr-label: meta/stale
           stale-pr-message: >
-            This PR has become stale because it has been open for 20 days without
-            activity. Remove the label and comment or this PR will be closed in 10 days.
+            This merge request has become stale because it has been open for 20 days without activity
+            Remove the label 'meta/stale' (if you're a maintainer) or comment on this merge request
+            (as an assignee). Otherwise, this merge request will be closed in 10 days automatically.
```

I'm only familiar with MR being used in Gitlab, while PR for Github.

I don't mind either way, but if you change from "pull request" wording to "merge request", it should be consistent. The `closed-pr-message` value still uses "PR".

Likewise the maintainer/assignee mentions are inconsistent with the phrasing for similar messages in this PR, that should be addressed. Might be worth mentioning the ignore label as another option for maintainers too?:

```suggestion
            This merge request has become stale because it has been open for 20 days without activity. This merge request will be closed in 10 days automatically unless:
            - A maintainer removes the label `meta/stale` label or adds the `stale-bot/ignore` label.
            - New activity occurs, such as a new comment.
```

---

Had been using GitLab before (and find the term "merge request" more fitting). Since GitHub uses pull requests, I will adjust this, right.